sigs.yaml

```diff
@@ -3375,6 +3375,54 @@ workinggroups:
     liaison:
       github: pohly
       name: Patrick Ohly
+- dir: wg-etcd-operator
+  name: etcd Operator
+  mission_statement: >
+    The working group is dedicated to enabling automatic and efficient operation of
+    etcd clusters in Kubernetes using an etcd-operator. The working group will discuss
+    the requirement and use cases of such an etcd-operator. It will also try to create
+    a roadmap to develop such an etcd-operator.
+
+    Note: the etcd clusters, to be managed by the etcd-operator, are to support applications
+    instead of Kubernetes itself.
+
+  charter_link: charter.md
+  stakeholder_sigs:
+  - Cluster Lifecycle
+  - etcd
+  label: etcd-operator
+  leadership:
+    chairs:
+    - github: ahrtr
+      name: Benjamin Wang
+      company: VMware
+    - github: hakman
+      name: Ciprian Hacman
+      company: Microsoft
+    - github: jberkus
+      name: Josh Berkus
+      company: Red Hat
+    - github: jmhbnz
+      name: James Blair
+      company: Red Hat
+    - github: justinsb
+      name: Justin Santa Barbara
+      company: Google
+  meetings:
+  - description: Regular WG Meeting
+    day: TBD
+    time: TBD
+    tz: PT (Pacific Time)
+    frequency: bi-weekly
+    url: https://zoom.us/my/cncfetcdproject
+    archive_url: provide-a-google-doc
```

Comment:
@jberkus could you please create a shared google doc in Kubernetes workspace, and share edit permission to both etcd-dev and cluster lifecycle?

Reply:
what's the name/format of the doc?  Is this for meeting notes, or something else?

Reply:
I think just we just need a google doc to meeting notes.

Reply:
@jberkus 

Name:  `Kubernetes etcd-operator Working Group Minutes & Agenda`

We just need to follow similar format as [wg-lts](https://docs.google.com/document/d/1RI_EL35MwQxrHqlWvtQNINhOSWergL3hmOSgC5PeZss/edit#heading=h.q66rfs3opd1m)

Reply:
Bit of a holdup with google drive permissions.  I'll get this created within then next couple days.

Reply:
See request here: #7937

Reply:
Thanks